docs/CONTRIBUTING

```diff
@@ -0,0 +1,41 @@
+# How to contribute
+
+I'm of the opinion that contributing should be as easy and inviting as possible. 
+This guide is written with that premise in mind.
+
+The discussion about Sessionless takes place on Discord in the [Open Source Force][osf] server. 
+Follow that link to get an invite. 
```

Comment:
There is a missing invitation link "https://opensourceforce.net/discord"

Reply:
The link is the Open Source Force above, which will actually be a link now that this is a md file